opsmngr/maintenance.go

```diff
@@ -0,0 +1,180 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package opsmngr
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	atlas "go.mongodb.org/atlas/mongodbatlas"
+)
+
+const (
+	maintenanceWindowsBasePath = "groups/%s/maintenanceWindows"
+)
+
+// MaintenanceWindowsService is an interface for interfacing with the Maintenance Windows
+// endpoints of the MongoDB Ops Manager API.
+// See more: https://docs.opsmanager.mongodb.com/current/reference/api/maintenance-windows/
```

Comment:
Please add an empty line so this can render properly
```suggestion
// endpoints of the MongoDB Ops Manager API.
//
// See more: https://docs.opsmanager.mongodb.com/current/reference/api/maintenance-windows/
```